packages/test-app-frontend/src/App/ITwinJsApp/ITwinJsApp.tsx

```diff
@@ -277,7 +277,24 @@ class MainFrontstageItemsProvider implements UiItemsProvider {
 
     return [{
       id: "ChangedElementsWidget",
-      content: <ChangedElementsWidget useV2Widget feedbackUrl="https://example.com" iModelConnection={UiFramework.getIModelConnection()!} />,
+      content: <ChangedElementsWidget useV2Widget
+        feedbackUrl="https://example.com"
+        iModelConnection={UiFramework.getIModelConnection()!}
+        manageNamedVersionsSlot={ManageNamedVersionsChild()}
```

Comment:
This is technically correct and does not cause any issues but it seems `ManageNamedVersionsChild` was intended to be a React component. If this is the case, this should instead read

`manageNamedVersionsSlot={<ManageNamedVersionsChild />}`